factor/fibonacci.py

```diff
@@ -5,5 +5,9 @@
 # 0, 1, 1, 2, 3, 5, 8 ...
 # ex: n = 6 -> 8
 def sequence(n: int) -> int:
-    # TODO implement
-    return 0
+    # this just sets up the fibonacci sequence.
+    if n == 0:
+        return 0
+    if n == 1:
+        return 1
+    return sequence(n-1)+sequence(n-2)
```

Comment:
watch style -- should be return sequence(n - 1) + sequence(n - 2) with proper spacing